vector/src/main/java/im/vector/app/features/location/LocationSharingViewModel.kt

```diff
@@ -70,6 +75,22 @@ class LocationSharingViewModel @AssistedInject constructor(
         setUserItem()
         updatePin()
         compareTargetAndUserLocation()
+        checkPowerLevelsForLiveLocationSharing()
+    }
+
+    private fun checkPowerLevelsForLiveLocationSharing() {
```

Comment:
Could be rename to `observePowerLevelsForLiveLocationSharing` since this is using a Flow.

Reply:
Thanks, done.